compiler/qsc_eval/src/intrinsic.rs

```diff
@@ -392,14 +392,14 @@ pub fn qubit_relabel(
             }
             (false, true) => {
                 // The right qubit has been relabeled, so we need to swap the left qubit with the
-                // qubit that the right qubit was relabeled to.
```

Comment:
Please consider an alternative version.
https://github.com/microsoft/qsharp/pull/2083
I believe it's easier to understand.

Reply:
I'll actually do a separate PR. This PR should be good as is.